libcxx/test/std/atomics/atomics.ref/fetch_add.pass.cpp

```diff
@@ -0,0 +1,114 @@
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+
```

Comment:
For the failures on picolib https://buildkite.com/llvm-project/libcxx-ci/builds/35217#018f401d-0d82-4d0a-8539-22802e30de78  I expect this fixes it.
```suggestion
// UNSUPPORTED: libcpp-has-no-threads

```

Reply:
Done.  Thank you.